bin/inx-chronicle/config.toml

```diff
@@ -1,5 +1,7 @@
 [mongodb]
 location = "mongodb://localhost:27017"
+username = "root"
+password = "pass"
```

Comment:
We shouldn't supply configs that don't work with MongoDB out of the box. Commenting these out would still signal the user the presence of these fields.

From a developers perspective, it might make sense to also look in `dotenv` for these values, that way we wouldn't have to ping pong back and forth because of different individual configurations.

```suggestion
# username = "root"
# password = "pass"
```

Reply:
Sure, but not having any username and password will also fail so 🤷